Packs/Slack/Integrations/Slack/Slack_test.py

```diff
@@ -311,6 +310,57 @@
 '''
 
 
+def test_exception_in_invite_to_mirrored_channel(mocker, capfd):
```

Comment:
avoid using `capfd` as we might miss stdout or stderr outputs, you should mock the relevant log functions instead